toffy/fov_watcher_test.py

```diff
@@ -26,9 +30,14 @@ def _slow_copy_sample_tissue_data(dest: str, delta: int = 10):
 
     print('copytime')
 
-    for tissue_file in os.listdir(TISSUE_DATA_PATH):
-        shutil.copy(os.path.join(TISSUE_DATA_PATH, tissue_file), dest)
+    for tissue_file in sorted(os.listdir(TISSUE_DATA_PATH)):
         time.sleep(delta)
+        if one_blank and '.bin' in tissue_file:
+            # create blank (0 size) file
+            open(os.path.join(dest, tissue_file), 'w').close()
+            one_blank = False
+        else:
+            shutil.copy(os.path.join(TISSUE_DATA_PATH, tissue_file), dest)
     print('copies done!')
```

Comment:
Same with this one

Reply:
oops, just some vestigial debugging code